install_config/install/advanced_install.adoc

```diff
@@ -270,6 +270,14 @@ use it.
 .2+.^|These variables are used to configure xref:../../install_config/certificate_customization.adoc#install-config-certificate-customization[custom certificates] which are deployed as part of the installation. See xref:advanced-install-custom-certificates[Configuring Custom Certificates] for more information.
 |`openshift_master_overwrite_named_certificates`
 
+|`openshift_hosted_router_create_certificate`
+|To provide a custom router certificate, set this parameter value to `False`.
```

Comment:
According to present code, when providing a custom router cert, not have to set its value to `False`.

Reply:
@jianlinliu, do you ever have to change this parameter value?

Reply:
@jianlinliu, if you never have to change the parameter, do we still need to describe it in this topic?

Reply:
At least according to my testing, I never set `openshift_hosted_router_create_certificate` in inventory file, customized router cert files are uploaded correctly. Maybe in some period, there is some existing installer bug, which force user to have to set this parameter to `False`, but now it is not a must. I think we do not need to describe it in this topic.